### PR TITLE
faml 745 rename psc-discrepancy-type to  psc-discrepancy-options

### PR DESCRIPTION
**Pull request by ICraig-CH**
- altered mention of psc-discrepancy-types to psc-discrepancy-options in all but the place where it sends to the api, it still needs to be changed. Fixed tests accordingly

Jira Ticket: https://companieshouse.atlassian.net/browse/FAML-745

### CODE REVIEW

**Review by ch-rfishwick — APPROVED** [review]

File: server/services/psc_discrepancy.js [hunk 1]
```diff
@@ -151,7 +151,7 @@ class PscDiscrepancy {
         details: data.details,
         psc_name: data.psc_name,
         psc_date_of_birth: data.psc_date_of_birth,
-        psc_discrepancy_types: data.psc_discrepancy_types
+        psc_discrepancy_types: data.psc_discrepancy_options
```
Comment (Zain-Abbas1) [inline server/services/psc_discrepancy.js:154]: Can we change this to psc_discrepancy_options

Comment (ICraig-CH) [reply server/services/psc_discrepancy.js:154]: that would break the api though

Comment (Zain-Abbas1) [reply server/services/psc_discrepancy.js:154]: Okay that's fine then I didn’t realise it was this one

**Review by Zain-Abbas1 — APPROVED** [review]

File: server/services/psc_discrepancy.js (hunk 1 above)
Comment (ICraig-CH) [reply server/services/psc_discrepancy.js:154]: sorry I should have mentioned in the description of the PR

**Pull request merged**